srcpkgs/txt2tags/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'txt2tags'
+pkgname=txt2tags
+version=2.6
+revision=1
+archs=noarch
+depends="python"
+short_desc="Converts text to various formats"
+maintainer="Thomas Batten <stenstorpmc@gmail.com>"
+license="GPL-2.0"
```

Comment:
License should be a SPDX-Identifier, so either `GPL-2.0-or-later` or `GPL-2.0-only`